web/src/stats/stats.ts

```diff
@@ -875,21 +977,33 @@ function populate_number_of_users(data) {
     });
 
     // Initial drawing of plot
-    draw_or_update_plot(all_time_trace, true);
+    // https://github.com/zulip/zulip/commit/fa9d79e203f83ab51b28e79bbb7a11589a2fd591
+    draw_or_update_plot(all_time_trace);
```

Comment:
This can be committed separately in a prep commit. Let's move this link to the commit message instead with some explanations.